irohad/consensus/gate_object.hpp

```diff
@@ -25,28 +26,33 @@ namespace iroha {
     struct PairValid {
       std::shared_ptr<shared_model::interface::Block> block;
       consensus::Round round;
+      std::shared_ptr<LedgerState> ledger_state;
```

Comment:
I think better to write explicit ctors for any gate_object. If this is unrelated changes, please, leave a todo instead.